circle.yml

```diff
@@ -5,3 +5,7 @@ machine:
 dependencies:
   override:
     - npm prune && npm install && ./node_modules/.bin/bower install
+
+test:
```

Comment:
we shouldn't need this command. our circleCI commands are configured for us, we ideally shouldn't override them unless we really have to

Reply:
If we delete it we get an issue with updating snapshots. https://circleci.com/gh/Financial-Times/kmt-header/223?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link - so definitely need this for now, but maybe issue need to be solved later on.... I don't know....